keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/DoubleTensor.java

```diff
@@ -59,6 +59,14 @@ static DoubleTensor zeros(int[] shape) {
         }
     }
 
+    static DoubleTensor linspace(double start, double end, int numberOfPoints) {
+        return Nd4jDoubleTensor.linspace(start, end, numberOfPoints);
+    }
+
+    static DoubleTensor arange(double start, double end) {
```

Comment:
Can we add a `step` or `interval` to this so that we can making things like:

`arange(0.0, 10.0, 2.0)` = `[0.0, 2.0, 4.0, 6.0, 8.0, 10.0]`?

https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.arange.html